asset/style/main.css

```diff
@@ -446,6 +470,39 @@ textarea {
   }
 
   .section-grid {
-    grid-template-columns: auto;
+    grid-template-columns: repeat(1, 1fr);
+  }
+}
+
+@media only screen and (min-width: 1021px) {
+  .form-container {
+    flex-direction: row;
+    justify-content: center;
+    align-items: flex-start;
+    width: 1171px;
+    height: 490px;
+    left: 135px;
+  }
```

Comment:
- [ ] [*Optional*] Both your about and contact form sections seem to be missing background images as you can see in the About section images below.

> Yours
![image](https://user-images.githubusercontent.com/90556221/166111644-7c5f0dc2-e24b-49a3-87cf-1526e5957b1a.png)

> Figma
![image](https://user-images.githubusercontent.com/90556221/166111682-8dc1ae8e-4bdf-441c-ba44-b3131642f7b3.png)

I've added this as optional since the reviewers seem to have missed it in the mobile version but please do make sure you add them whenever you get the chance 👍